### PR TITLE
chores/rubocop reduce complexity #160158784

### PR DESCRIPTION
**Pull request by akegan**
Note: This PR actually fixes a bug where lease up status update date field was not pulling from the correct place in the salesforce object in addition to fixing the issue of rubocop reporting a function with too high a complexity. 
https://www.pivotaltracker.com/story/show/160158784

### CODE REVIEW

File: spec/javascript/components/lease_ups/LeaseUpsPage.test.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ describe('LeaseUpApplicationsPage', () => {
 
   test('Should render LeaseUpTable', () => {
     const applications = [ buildApplicationPreference(1), buildApplicationPreference(2) ]
-
+    console.log("PLEASE LOG THIS", applications)
```
Comment (fddayan) [inline spec/javascript/components/lease_ups/LeaseUpsPage.test.js]: @akegan the message is pretty clear, but do we want this console.log in here? why?

**Review by fddayan — APPROVED** [review]
Looks good to me, I just added one comment. It looks like you are logging that intentionally.

File: spec/javascript/components/lease_ups/LeaseUpsPage.test.js (hunk 1 above)
Comment (akegan) [reply spec/javascript/components/lease_ups/LeaseUpsPage.test.js]: nope, it's just a debug. I'll remove it

Comment (akegan) [reply spec/javascript/components/lease_ups/LeaseUpsPage.test.js]: I was just really hoping that I would see the log 🤣

**Pull request merged**